src/pages/Login.tsx

```diff
@@ -0,0 +1,69 @@
+import { useLoginPwForm } from '@/hooks/useLoginPwForm';
+import NavigationBar from '@components/NavigationBar';
+import { useLoginEmailForm } from '@hooks/useLoginEmailForm';
+import { StyeldLoginInput } from '@styles/Login/StyeldLoginInput';
+import { StyledLoginButton } from '@styles/Login/StyledLoginButton';
+import { StyledLoginComponentContainerDiv } from '@styles/Login/StyledLoginComponentContainerDiv';
+import { StyledLoginComponentDiv } from '@styles/Login/StyledLoginComponentDiv';
+import { StyledLoginKakoLogo } from '@styles/Login/StyledLoginKakoLogo';
+import { useMemo } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+const Login: React.FC = () => {
+  const navigate = useNavigate();
+
+  const handelBackOrHome = () => {
+    if (window.history.length > 2) {
+      navigate(-1);
+    } else {
+      navigate('/');
+    }
+  };
+
+  const { id, idError, handleIdBlur, handleIdChange } = useLoginEmailForm();
+  const { pw, pwError, handlePwBlur, handlePwChange } = useLoginPwForm();
+
+  // 로그인 버튼 활성화 조건 (기존과 동일)
+  const isLoginButtonEnabled = useMemo(() => {
+    const isIdValid = !id || idError ? false : true;
```

Comment:
아래처럼 작성해 볼 수 있을거 같네요

```suggestion
    const isIdValid = id || !idError;
```